CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+`0.7.1 <https://github.com/SwissDataScienceCenter/renku-gateway/compare/0.7.0...0.7.1>`__ (2020-04-24)
+------------------------------------------------------------------------------------------------------
+
+Features
+~~~~~~~~
+
+-  add routing and "login" route for anonymous notebooks sessions.
+   (`#193 <https://github.com/SwissDataScienceCenter/renku-gateway/issues/193>`__ ,
+   `#195 <https://github.com/SwissDataScienceCenter/renku-gateway/commit/ee94e63bab0d3e70cf2cdc23f12df1faf50c9592>`__),
```

Comment:
you wanted to link the issue #195 (then wrong link, should be `https://github.com/SwissDataScienceCenter/renku-gateway/issues/195`) or the commit ee94e63 (then wrong text, should be `ee94e63`)?

Reply:
oops, thanks for paying attention!